homeassistant/components/ovo_energy/config_flow.py

```diff
@@ -6,11 +6,11 @@
 from homeassistant.config_entries import ConfigFlow
 from homeassistant.const import CONF_PASSWORD, CONF_USERNAME
 
-from .const import DOMAIN
+from .const import ACCOUNT, DOMAIN
```

Comment:
CONF_ACCOUNT

Reply:
is that the home assistant suggested naming